training/finetune_acft_hf_nlp/components/pipeline_components/question_answering/spec.yaml

```diff
@@ -6,27 +6,26 @@ display_name: Question Answering Pipeline
 description: Pipeline Component to finetune Hugging Face pretrained models for extractive question answering task. The component supports optimizations such as LoRA, Deepspeed and ONNXRuntime for performance enhancement. See [docs](https://aka.ms/azureml/components/question_answering_pipeline) to learn more.
 
 inputs:
-  # Compute parameters
-  compute_model_import:
+  instance_type_model_import:
     type: string
-    optional: false
-    description: compute to be used for model_import eg. provide 'FT-Cluster' if
-      your compute is named 'FT-Cluster'. Special characters like \ and ' are invalid in the parameter value.
-  compute_preprocess:
+    optional: true
+    default: Standard_d12_v2
+    description: Instance type to be used for model_import component in case of serverless compute, eg. standard_d12_v2
```

Comment:
Can you make it explicit that the parameter compute_name_model_import needs to be set to serverless for this to be considered